bindings/iDynTree.i

```diff
@@ -350,6 +353,8 @@ TEMPLATE_WRAP_MOTION_FORCE(ForceVector3, WRAP_FORCE, SET_NAME_FOR_WRAPPER,,)
 // High level interfaces
 %include "iDynTree/KinDynComputations.h"
 
+%template(MatrixVector) std::vector<iDynTree::Matrix4x4>;
```

Comment:
This is ambiguous, how we would call then a Vector of Matrix of different sizes?  I suggest to call  this `Matrix4x4Vector` .